spec/like_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Likes, type: :model do
+  before(:all) do
+    @user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.',
+                        posts_counter: 0)
+    @post = Post.create(author: @user, title: 'Post communication', text: 'This is my first post', likes_counter: 0,
+                        comments_counter: 0)
+  end
+
+  context 'Associations' do
+    it 'belongs to an author' do
+      like = Likes.reflect_on_association('author')
+      expect(like.macro).to eq(:belongs_to)
+    end
+
+    it 'belongs to a post' do
+      like = Likes.reflect_on_association('post')
```

Comment:
- [x] Same case here.

```suggestion
      like = Like.reflect_on_association('post')
```

Reply:
hy @Lameck1 Thank you for helping much appreciated 🙏 
Changes were made as per your request 👍